src/App.jsx

```diff
@@ -9,12 +9,17 @@ import {
   DialogTitle,
   DialogContent,
 } from "@mui/material";
-import { makeStyles } from "@mui/styles";
```

Comment:
$ git pull
fatal: unable to access 'https://github.com/narayan954/dummygram.git/': Failed to connect to github.com port 443 after 18973 ms: Timed out

I don't know why but I am getting this issue. many times.  whenever I try to pull the repo before push. Even I am not able to push the code.

Reply:
It's because you haven't set your remote, just google this error, 
tldr - You're trying to directly push to my repo instead of your copy of my repo

Reply:
> It might be because you haven't set your remote, just google to set remote properly,

tldr - You're trying to directly push to my repo instead of your copy of my repo

Reply:
but I have added remote origin

Reply:
![image](https://user-images.githubusercontent.com/78996216/208491011-a17fbe48-a8e9-48eb-b442-283eb6fa32f8.png)

Reply:
![image](https://user-images.githubusercontent.com/78996216/208491155-7cb84897-8996-42b9-b46d-f0b1ecf9d2e6.png)


And How to resolve these.

Reply:
that's fake positive don't worry about that

Reply:
> but I have added remote origin

yeah, that's not because of git push but git pull

Reply:
> Failed to connect to github.com port 443 after 18973 ms: Timed out

try https://stackoverflow.com/questions/18356502/github-failed-to-connect-to-github-443-windows-failed-to-connect-to-github